pkgs/development/python-modules/usb-monitor/default.nix

```
@@ -0,0 +1,29 @@
+{
+  buildPythonPackage,
+  fetchPypi,
+  lib,
+  python3Packages,
+}:
+
+buildPythonPackage rec {
+  pname = "usb-monitor";
+  version = "1.21";
+
+  src = fetchPypi {
+    inherit version;
+    pname = "usb_monitor";
+    hash = "sha256-M+BUmbNxQWcULFECexTnp55EZiJ6y3bYCEtSwqKldAk=";
+  };
+
+  dependencies = [ python3Packages.pyudev ];
```

```suggestion
  dependencies = [ pyudev ];
```
otherwise we mix python versions